tests/unit/test-extension-runners/test.extension-runners.js

```diff
@@ -488,7 +488,8 @@ describe('util/extension-runners', () => {
     it('can reload when user presses R in shell console', async () => {
       const {extensionRunner, reloadStrategy} = prepare();
 
-      const fakeStdin = new tty.ReadStream();
+      // $FLOW_FIXME: flow doesn't yet recognize 0 as a valid parameter of tty.ReadStream.
+      const fakeStdin = new tty.ReadStream(0);
```

Comment:
Wouldn't it be easier to maintain if you create a new `FakeReadableStream` or `FakeStdin` class?

Reply:
:+1: good point, let's move it into the test helpers module.